README.md

```diff
@@ -44,65 +45,106 @@ Usage instructions are available in the bootstrap [README](./0-bootstrap/README.
 
 ### [1. org](./1-org/)
 
-The purpose of this stage is to set up top level folders used to house projects which contain shared resources such as monitoring, networking, org level logging and also to set baseline security settings through organizational policy.
+The purpose of this stage is to set up the common folder used to house projects which contain shared resources such as DNS Hub, Interconnect, SCC Notification, org level secrets and org level logging.
 This will create the following folder & project structure:
 
 ```
 example-organization
 └── common
-    ├── logs
-    │   ├── org-audit-logs
-    │   └── org-billing-logs
-    ├── monitoring
-    │   ├── org-monitoring-nonprod
-    │   └── org-monitoring-prod
-    └── networking
-        ├── org-shared-vpc-nonprod
-        └── org-shared-vpc-prod
+    ├── prj-p-org-audit-logs
+    ├── prj-p-org-billing-logs
+    ├── prj-p-org-dns-hub
+    ├── prj-p-org-interconnect
+    ├── prj-p-org-scc
+    └── prj-p-org-secrets
 ```
 
 #### Logs
 
-Under the logs folder, two projects are created. One for organization wide audit logs and another for billing logs.
+Under the common folder, two projects are created. One for organization wide audit logs and another for billing logs.
 In both cases the logs are collected into BigQuery datasets which can then be used general querying, dashboarding & reporting.
 For the various audit log types being captured in BigQuery, this is mirroring the standard [retention periods](https://cloud.google.com/logging/quotas#logs_retention_periods) for Cloud Logging, these settings can be adjusted according to your requirements.
 
 For billing data, a BigQuery dataset is created with permissions attached however you will need to configure a billing export [manually](https://cloud.google.com/billing/docs/how-to/export-data-bigquery) as there is no easy way to automate this currently.
 
+#### DNS Hub
+
+Under the common folder, one project is created. This project will host the DNS Hub for the organization.
+
+#### Interconnect
+
+Under the common folder, one project is created. This project will host the Interconnect infrastructure for the organization.
+
+#### SCC Notification
+
+Under the common folder, one project is created. This project will host the SCC Notification infra structure at organization level.
+This project will contain a Pub/Sub topic and subscription, a SCC Notification configured to send all new Findings to the topic created.
+
+
+#### Secrets
+
+Under the common folder, one project is created. This project will host the Secret Manager for secrets shared by the organization.
```

Comment:
I would replace "will host the Secret Manager" with "is allocated for GCP Secret Manager"